ams/optimizer/src/main/java/com/netease/arctic/optimizer/KubernetesOptimizerContainer.java

```diff
@@ -0,0 +1,89 @@
+      "    client-key-data: LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFb3dJQkFBS0NBUUVBdmFyOUJIekpPcTl6MmFaQ3VRNUhNTHBCaTBueDEwT3lCRzhKbGFFTUZWSzJSZndDClVjaHdqTzh0OTdDZVN0ZEVBWUpQWUlkanovSGZPdFFvUFZQM1RnOTBzdFJENWlPSnNvVjRMMi9peGlDcThyWGoKUU9qaGN4QU1wQnpGT3luR3VNYnp0dEt5WVRESVowbnJXb0ljT1Bid3c5K1ZvaFhhVW9QdWY4b2xWZms1TE9UKwp5ZitpbkV2eWFaUGZ5TkRLRVBBMlV5cW9URWF5Y2NkUjZkTmdlWUNFT0NoK2dpWnFVa0FDMjhZNEVVVmpIbkF6Ckd1Tnp4dGdQV2ltTHBqbEpjaW1yTTRPZUl4Q1FzZDBhc2RRSnVGWFlEaFBIa1czNVNJbDk0OGtueDJ0TWhHWnQKaDVDTjdXTDFLL3BtQXptWk1mWGpoSDFPdVhsa25CMktVVnZialFJREFRQUJBb0lCQUNRUUYrMjdnRk45T3N6ZQpkUDlVdktxQ0w2WTVXQmR6RXEwUEk4WmtpYlNnTm5JV0dhYk5Nc0ZKVlBjc2lOeDRFOEVwc3NnSzFpcWF0YlFzCjFMM2Nja0JRWmdMK295NW1BVytGT3pYaDB6K1N4STVEa1VNdFJIaXBTNDRFdm1laWFOdUhVSjJwY0N0VXFEWWoKY3ZHUm5hWWpKZUpJWjk0YXc1ays1cUU3b1YrNElHcjdETUdUM2dETk9MamZMS3E5Zjl1ZFREUkJwZkltbFdPNQpLU0d5b0xFSlJXSkd0SDUxdkV4aWN6TUMzdzhKNnFWSkUvQ09jamdSV3VpMEMrVGZtMUdUcjUwSk1VYUtiTGhyClpFYUMwUEhETVVRMmxzUUVFbmMwRmJvZmRxcytvMTFndDh6QjlEUDNHVzdCSjd3QkNwNlRNRm5DWk5DUlZZNWQKeUlyWnIwRUNnWUVBelZuWktHVThGOXZROFRrTmdHOHQxaDIvRURYODZ5UlZDeVhmQTBuaVZOUFpJVEQrRGM1QQozbkp4M052eWNnT1l4U0ttVkgzYUU0UDJpSUtWMnR2bEdFM1h3VjBxbUM5ditaVFBZdjZBVnQ5bmxjcllGWUV0CkRnb2xrTnUvQzZydUtDbG1zeGtheXcyR3drSTlLTmtBREhEWHhyM0FoRlltMEpGRHQ2NDRwOTBDZ1lFQTdITGoKNTdoTnlyWmpneEZDUnp1dUNOSUgxcU5WckNaUFlwYzdPZURONHZURkd3SjYyc1M4OU1wY3hpd1Q2UytJRGh2cwoyZitlckxIK0FGYTBaN0l6SnpIYndxcFlwYVY3MU45THRKUnRQd0lFSGY1MFcwc1lZQ0pMbFNPMVhxT1ozdVZDClR0cFNVczdkTFVRMERsSTFUZy9JOENmVmlNL2Y0TWZld1ZpK0gzRUNnWUJQbFkyeXVTRkVBZDRGVHQ0cnMycnAKTzVnTHVWQ3U5T0s4c2sydTRaaUUxYUdsMm0zcmZjN29KeVIzdXdwSUk1cTJkQXBRWG9JQTVEak1pUWQ0elpZSgpDRW9nMTNHbGoyVHZManY5bXJLMGVGcVYxQXBRczBKNTJYYmJvRDUzVUNTQ2ppRU9NaUdQSmt2ZXgzc2FkSmN2Ck95QjFGcDhnNnA2YVlHSUZNdEVrUlFLQmdRQ3ZvMkJiL25INnhLVUM5VTBRY09xRUx0QVh4bGliZWhHNklMQ2oKKzdPMGhUSHRNRmhtTFlKWExBTGlTbGUzL2RESStrRmtaaGRPSFNHYXlzMVR3ZkZ4aWYyK2lwOHkzTXd4Z25WUAovSGx5Tm1Nc2pKbU9QeWdxTVErSUIzQndqb0o4S2p5cEtrL0FwMTF3aEp0T2tBNThvQWtaSzkzWXRPR09yYWx3CllpVklZUUtCZ0ZsMEN0RlhRZmlHN2VubzdsRWVlV3dSRXFxL01vZGVPcjRtRUZJWk84VEZwUHlpK1NuRklRUVEKQWdQRW5SdnZtb29IM1VDTzNRaUU3UHA5VzdrTTd6ZG1IZ09RTWNZK1NLMkdqU21BYzJXQWlGWkM0aXduTkV6QQp1bE5wLzh4d1BtbS9ZTGwyQTR3ckh4ZE5CQkhmRUlhMDZha21TbmRVTS9Uck9vQjVCTG1KCi0tLS0tRU5EIFJTQSBQUklWQVRFIEtFWS0tLS0tCg==";
+
+
+  @Override
+  protected Map<String, String> doScaleOut(String startUpArgs) {
+    try {
+      String[] cmd = { startUpArgs};
+      LOG.info("Starting local optimizer using command : {}", startUpArgs);
+      Config config = Config.fromKubeconfig(KUBECONFIG_CONTENTS);
+      KubernetesClient client = new KubernetesClientBuilder().withConfig(config).build();
+      K8sUtil.runJob("default","amoro-optimizer",
+          client, "registry.cn-hangzhou.aliyuncs.com/udh/amoro-optimizer:0.6.0",String.join(" ",cmd),LOG);
```

Comment:
The image address can be passed in through the properties of OptimizerContainer.

Reply:
Some information related to the K8S cluster can be configured in the K8S Properties, such as the kube_config address and service_account.